.travis.yml

```diff
@@ -65,6 +65,10 @@ matrix:
       env: SONATA_ADMIN='dev-master as 3.x-dev'
     - php: '7.3'
       env: SYMFONY_DEPRECATIONS_HELPER=0
+    - php: '7.3'
```

Comment:
FYI: You should update this inside the dev-kit repo or this will be reverted the following day